### PR TITLE
[asl] Fence instructions

### PR DESCRIPTION
**Pull request by maranget**
Implementation is quite straightforward: emit appropriate events.

### CODE REVIEW

File: herd/libdir/asl-pseudocode/implementations.asl [hunk 1]
```diff
@@ -609,3 +609,55 @@ getter SCTLR_EL1[] => SCTLRType
 begin
   return Zeros(64);
 end
+
+// InstructionSynchronizationBarrier()
+// ===================================
+func InstructionSynchronizationBarrier()
+begin
+  primitive_isb();
+  return;
```
Comment (HadrienRenaud) [inline herd/libdir/asl-pseudocode/implementations.asl:618]: FYI, `return;` is not mandatory when not returning anything in ASLv1

**Review by HadrienRenaud — COMMENTED** [review]
Hi @maranget 

Very cool that you managed to implement fences 🎉

File: herd/libdir/asl-pseudocode/implementations.asl (hunk 1 above)
Comment (maranget) [reply herd/libdir/asl-pseudocode/implementations.asl:618]: Thanks.

**Comment by maranget** [comment]
Merged, thanks @HadrienRenaud for the review.

**Pull request merged**